robocopkaka/event-manager-rails
# Feat users can signup and login ui 162905507

#### What does this PR do?
Allow users signup and login
#### Description of Task to be completed?
* Allow users signup
* Allow users login
#### How should this be manually tested?
* Start the application
* Visit `http://{baseURL}/signup`
* You should see a signup form
* Enter the relevant details and click the submit button
* You should see a toast informing you of the status of your request and the fields should be cleared
* Visit `http://{baseURL}/login`
* You should see a login form
* Enter the relevant details and click the submit button
* You should see a toast informing you of the status of your request and the fields should be cleared
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories? (if applicable)
[#162905507](https://www.pivotaltracker.com/story/show/162905507)
